### PR TITLE
Vinhng Fix Bug Change Password

### PR DESCRIPTION
**Pull request by vinh1332002**

### CODE REVIEW

File: frontend/src/Constants/ErrorMessages.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const PASSWORD_AT_LEAST_ONE_LOWERCASE =
 export const PASSWORD_AT_LEAST_ONE_UPPERCASE =
   "Please input at least one upper case letter!";
 export const PASSWORD_AT_LEAST_ONE_SPECIAL_CHARACTER =
-  "Please input at least one special character!";
+  "Please input at least one special character without space and unsported special character!";
 export const PASSWORD_RANGE_FROM_8_TO_16_CHARACTERS =
```
Comment (truongtrinhv) [inline frontend/src/Constants/ErrorMessages.js]: Typo: This string should be: Please input at least one special character without space and the unsupported special characters!
Please double check

File: frontend/src/Constants/ErrorMessages.js [hunk 2]
```diff
@@ -15,3 +15,4 @@ export const INCORRECT_OLD_PASSWORD =
   "Old password is incorrect. Please try again!";
 export const PASSWORD_COMPARED =
   "The new password cannot match the old password!";
+export const PASSWORD_ONLY_ALLOW = "Please enter valid value!";
```
Comment (truongtrinhv) [inline frontend/src/Constants/ErrorMessages.js]: Typo- This string should be: Please enter a valid value!

File: frontend/src/Pages/AdminPage/ManageUserPage/CreateUserPage.jsx [hunk 3]
```diff
@@ -21,23 +24,33 @@ export function CreateUserPage() {
...
   };
 
-  const dateFormatList = ["DD/MM/YYYY", "DD/MM/YY"];
+  const dateFormatList = ["YYYY/MM/DD", "YYYY/MM/DD"];
 
   const layout = {
-    labelCol: { span: 2 },
-    wrapperCol: { span: 5 },
+    labelCol: {
+      span: 2,
```
Comment (truongtrinhv) [inline frontend/src/Pages/AdminPage/ManageUserPage/CreateUserPage.jsx]: Redudant comma

File: frontend/src/Pages/AdminPage/ManageUserPage/CreateUserPage.jsx [hunk 4]
```diff
@@ -21,23 +24,33 @@ export function CreateUserPage() {
...
+    },
+    wrapperCol: {
+      span: 5,
```
Comment (truongtrinhv) [inline frontend/src/Pages/AdminPage/ManageUserPage/CreateUserPage.jsx]: Redudant comma

File: frontend/src/Pages/AdminPage/ManageUserPage/CreateUserPage.jsx [hunk 5]
```diff
@@ -21,23 +24,33 @@ export function CreateUserPage() {
...
+    },
   };
 
   const tailLayout = {
-    wrapperCol: { offset: 3 },
+    wrapperCol: {
+      offset: 3,
```
Comment (truongtrinhv) [inline frontend/src/Pages/AdminPage/ManageUserPage/CreateUserPage.jsx]: Redudant comma

File: frontend/src/Pages/AdminPage/ManageUserPage/ManageUserPage.jsx [hunk 6]
```diff
@@ -1,7 +1,10 @@
+import { Outlet } from "react-router-dom";
+
 export function ManageUserPage() {
   return (
     <>
-      <h1>User Management Page</h1>
+      {/* <h1>User Management Page</h1> */}
```
Comment (truongtrinhv) [inline frontend/src/Pages/AdminPage/ManageUserPage/ManageUserPage.jsx]: Remove comment code

File: frontend/src/Pages/AuthenticationPage/ChangePasswordPage.jsx [hunk 7]
```diff
@@ -85,6 +86,11 @@ export function ChangePasswordPage() {
                 required: true,
                 message: PASSWORD_REQUIRED,
               },
+              {
+                min: 8,
+                max: 16,
+                message: PASSWORD_RANGE_FROM_8_TO_16_CHARACTERS,
```
Comment (truongtrinhv) [inline frontend/src/Pages/AuthenticationPage/ChangePasswordPage.jsx]: Redundant comma

File: frontend/src/Pages/AuthenticationPage/LoginPage.jsx [hunk 8]
```diff
@@ -73,6 +74,11 @@ export function LoginPage() {
                 required: true,
                 message: PASSWORD_REQUIRED,
               },
+              {
+                pattern:
+                  /^(?=.*[A-Za-z0-9])[A-Za-z0-9!*_@#$%^&+=<>|.,:;"'{})(-?/`~]*$/,
+                message: PASSWORD_ONLY_ALLOW,
```
Comment (truongtrinhv) [inline frontend/src/Pages/AuthenticationPage/LoginPage.jsx]: Redudant comma

File: frontend/src/App.jsx [hunk 9]
```diff
@@ -27,11 +27,13 @@ function App() {
...
-            path="/admin/manage-user/create-user"
-            element={<CreateUserPage />}
-          />
+          <Route path="/admin/manage-user" element={<ManageUserPage />}>
+            <Route
+              path="/admin/manage-user/create-user"
+              element={<CreateUserPage />}
+            />
+          </Route>
+
```
Comment (truongtrinhv) [inline frontend/src/App.jsx]: Redudant empty line

File: frontend/src/Pages/AuthenticationPage/ChangePasswordPage.jsx (hunk 7 above)
Comment (vinh1332002) [reply frontend/src/Pages/AuthenticationPage/ChangePasswordPage.jsx]: Redundant comma is auto generated by Format code Prettier

File: frontend/src/Pages/AuthenticationPage/LoginPage.jsx (hunk 8 above)
Comment (vinh1332002) [reply frontend/src/Pages/AuthenticationPage/LoginPage.jsx]: Redundant comma is auto generated by Format code Prettier

File: backend/API/appsettings.Development.json [hunk 10]
```diff
@@ -6,6 +6,6 @@
     }
   },
     "ConnectionStrings": {
-        "RookiesConnectionString": "Server=localhost\\MSSQLSERVER2019;Database=TheRookiesDB;Trusted_Connection=True;"
+        "RookiesConnectionString": "Server=localhost\\SQLExpress;Database=TheRookiesDB;Trusted_Connection=True;"
```
Comment (truongtrinhv) [inline backend/API/appsettings.Development.json:9]: No check in DEV setting

**Review by truongtrinhv — APPROVED** [review]

**Pull request merged**